packages/react/src/definitions/kyc.ts

```diff
@@ -185,6 +185,30 @@ export interface KycLegalEntityData {
   legalEntity: LegalEntity;
 }
 
+export enum DocumentTypes {
+  IDCARD = 'ID Card',
+  PASSPORT = 'Passport',
+  DRIVERS_LICENSE = "Driver's License",
+  RESIDENCE_PERMIT = 'Residence Permit',
+}
+
+export enum GenderTypes {
+  MALE = 'Male',
+  FEMALE = 'Female',
+}
```

Comment:
Enums normally have a name in singular (e.g. `DocumentType` instead of `DocumentTypes`)